app/views/items/show.html.erb

```diff
@@ -1,26 +1,27 @@
 <% content_for :sidebar do %>
-  <%= link_to 'Return to Listing', items_path, :class => 'btn' %></br></br>
-  <%= edit_item_button %></br></br>
-  <%= add_repair_button %></br></br>
-  <%= add_inspection_button %></br></br>
+  <%= link_to 'Return to Listing', items_path, class: 'btn btn-primary', style: 'width: 100%' %><br/><br/>
+  <%= edit_item_button %><br/><br/>
+  <%= add_repair_button %><br/><br/>
+  <%= add_inspection_button %><br/><br/>
 <% end %>
-<p>
-  <h3>
-    <span class="badge <%= 'badge-success' if @item.status == 'Available' %>">
-      <%= @item.status %>
-    </span>&nbsp;
-    <%= @item.name + " ( " + @item.brand + " " + @item.model + " )" %></br>
-  </h3>
-</p>
+
+<h3>
+  <span class="label <%= item_label(@item) %>">
+    <%= @item.status %>
+  </span> &nbsp;
+  <%= @item.name + " ( " + @item.brand + " " + @item.model + " )" %>
+</h3>
 
 <% if @item.description.present? %>
   <p><b>Description:</b> <%= @item.description %></p>
 <% end %>
 
-<p><b>Location:</b> <%= @item.location %></p>
+<% if @item.location.present? %>
+  <p><b>Location:</b> <%= @item.location %></p>
+<% end %>
 
 <% if @item.source.present? %>
-  <p>Source:<b><%= @item.source %></b></p>
+  <p><b>Source:</b> <%= @item.source %></p>
```

Comment:
It's no big deal, but I typically bold the content, not the labels. After a few times using it, people will be much more interested in the content, and will know where it lives.